api/models/user.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const userSchema = Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  login: {
+    type: String,
+    required: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  favorites: {
+    type: [String],
```

Comment:
If that's about liked posts, better ref to Likes to Posts

Reply:
Fixed